site/sfguides/src/tempo/tempo.md

```diff
@@ -0,0 +1,228 @@
+author: mando222
+id: tempo
```

Comment:
@mando222 The title should be match the title in lowercase with underscores instead of spaces.

Reply:
Fixed